Packages/Models/Sources/Models/Comment.swift

```diff
@@ -11,7 +11,7 @@ public struct Comment: Codable {
     public let url: String
     public let id: Int?
     public let nodeID: String?
-    public let user: User
+    public let user: User?
```

Comment:
Handle case if the user is deleted


https://github.com/ldakhoa/GistHub/assets/39718754/c8086cef-017e-42a9-93a8-ab8bba54fbfd